lib/urlwatch/jobs.py

```diff
@@ -230,3 +231,29 @@ def retrieve(self, job_state):
                 return response.content.decode('ascii', 'ignore')
 
         return response.text
+
+class FileJob(Job):
+    """Read the content of a local file"""
+
+    __kind__ = 'file'
+    __required__ = ('path',)
+
+    def get_location(self):
+        return str(self.path)
+
+    def retrieve(self, job_state):
+
+        path_string = str(self.path)
+
+        if not path_string:
+            raise ValueError('Path cannot be null or empty')
+
+        if path_string.startswith('file://'):
```

Comment:
Maybe I don't fully get your proposal, since to me It seems a kludge that won't be helpful if some user decide to use alternative protocols such as `ftp://`. Am I right? Can you try to clarify?